services/hooks/src/listeners.js

```diff
@@ -203,9 +204,9 @@ class HookListeners {
    */
   async _reconcileConsumers() {
     let queues = [];
-    await this.Queues.scan({}, {
-      limit: 1000,
-      handler: (queue) => queues.push(queue)});
+    await queueUtils.getHooksQueues(this.db, queue => {
+      queues.push(queue);
+    });
```

Comment:
Since this just loads the entire table into memory, this could all be simplified to an un-paginated call to `db.fns.get_hooks_queues`.  The queues table is generally pretty small (strictly smaller than the list of hooks), and it's been this way for a long time, so I think this is OK for now, and makes the rest of this reconciliation simpler.